### ISSUE TITLE
Error: is not a valid chart repository or cannot be reached despite valid repo

### ISSUE THREAD

**Issue by bitsofinfo** [aslafy-z/helm-git#13]
I can't seem to get this to work. I can git clone this repo locally w/ no issues

```
bash-4.4$ helm version
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}

bash-4.4$ git version
git version 2.16.2

bash-4.4$ helm repo --debug add my-app-base git+ssh://git@git.mydomain.com/my-apps-helm/my-app-base@charts
Warning in plugin 'helm-git': git_ref is empty, defaulted to 'master'. Prefer to pin GIT ref in URI.
fatal: No path specified. See 'man git-pull' for valid url syntax
Error in plugin 'helm-git': error Unable to sparse-checkout. Check your Git ref (master) and path (charts).
Error: Looks like "git+ssh://git@git.mydomain.com/my-apps-helm/my-app-base@charts" is not a valid chart repository or cannot be reached: plugin "helm-git" exited with error
```

**Comment by bitsofinfo** [comment]
any thoughts on what is going on here? My url looks legit no?

**Comment by aslafy-z** [comment]
#14 should fix it. Thanks for reporting!

**Comment by bitsofinfo** [comment]
thanks, can u please cut a release?

**Comment by aslafy-z** [comment]
Just released [`v0.4.1`](https://github.com/aslafy-z/helm-git/releases/tag/v0.4.1) that includes the fix!

**Comment by bitsofinfo** [comment]
thanks
